test/ctia/test_helpers/es.clj

```diff
@@ -134,3 +155,28 @@
 
 (defn post-all-to-es [objects]
   (run! post-to-es objects))
+
+(defn str->edn
```

Comment:
the naming of the function is not totally accurate, the final result is not an edn string